public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/AlertManagerRouting.tsx

```diff
@@ -22,11 +22,22 @@ export function AlertManagerManualRouting({ alertManager }: AlertManagerManualRo
   const styles = useStyles2(getStyles);
 
   const alertManagerName = alertManager.name;
-  const { isLoading, error: errorInContactPointStatus, contactPoints } = useContactPointsWithStatus();
+  const { isLoading, error: errorInContactPointStatus, contactPoints, refetchReceivers } = useContactPointsWithStatus();
   const [selectedContactPointWithMetadata, setSelectedContactPointWithMetadata] = useState<
     ContactPointWithMetadata | undefined
   >();
 
+  const [loadingContactPoints, setLoadingContactPoints] = useState(false);
+
+  const onClickRefresh = () => {
+    refetchReceivers();
+    // show loading spinner for 1 second
+    setLoadingContactPoints(true);
+    setTimeout(() => {
+      setLoadingContactPoints(false);
+    }, 1000);
+  };
```

Comment:
we need to simulate this loading state as sometimes this is too fast and user does not see any feedback.

Reply:
Yes, I think we should! I've just pushed a commit clearing this timeout 👍